src/components/notification/index.tsx

```diff
@@ -230,10 +230,7 @@ const ViewNotificationItem: React.FC<NotificationItemProps> = ({
           <Pool>
             <PoolShare theme = {theme}>
               {timeStamp
-                ? moment
-                    .utc(parseInt(timeStamp) * 1000)
-                    .local()
-                    .format("DD MMM YYYY | hh:mm A")
+                ? format(new Date(Number(timeStamp) * 1000), 'dd MMM yyyy | hh:mm a')
```

Comment:
can we paste some screenshot of the old date string vs what we get with this transformation?